### PR TITLE
KMA Tokenomics Update

### PR DESCRIPTION
**Pull request by Garandor**

### CODE REVIEW

File: docs/calamari/KMA/01-Overview.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Overview
+
+KMA is the first community-driven deflationary native token of Calamari Network, the Kusama-deployed canary-net of Manta Network. At genesis in Nov 2021 an initial supply of 10 billion KMA tokens was created. Around 24% of this supply was distributed to crowdloan participants as part of the rewards for helping Calamari Network secure the first parachain slot. Furthermore, KMA does not have any private sales or team tokens.
+
+The KMA token will has multiple uses on Calamari Network:
```
Comment (grenade) [inline docs/calamari/KMA/01-Overview.md:5]: `will have` or `has` (without `will`)

File: docs/calamari/KMA/01-Overview.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+1. Every transaction on Calamari (including MantaPay on Calamari) collects a Network Usage Fee in the form of KMA token. This network fee is split three ways:
+    - 45% will be burned.  
+    - 45% is sent to the treasury to fund future parachain slot auctions and community expenses like a development fund.
+    - 10% is distributed to the collator who produced the block containing this transaction.
+2. The KMA token serves as a medium of exchange.
+3. The KMA token represents governance rights on Calamari.
+4. MantaPay on Calamari public-to-private and vice versa transactions will not charge any additional premium for now. 
+5. To incentivize decentralization, for the first 3 years after launch 3% of total KMA issuance will be distributed as staking rewards per year by minting of new KMA.
+All newly minted KMA this way will periodically be burned from the treasury to compensate this inflation. After 3 years, the Calamari community will decide on further incentives.
```
Comment (grenade) [inline docs/calamari/KMA/01-Overview.md:14]: to compensate `for` this inflation

**Review by grenade — DISMISSED** [review]
lgtm

File: docs/calamari/KMA/01-Overview.md (hunk 2 above)
Comment (Garandor) [reply docs/calamari/KMA/01-Overview.md]: addressed, thx!

**Review by Kevingislason — DISMISSED** [review]

**Review by grenade — APPROVED** [review]

**Pull request merged**